package.json

```diff
@@ -62,12 +62,13 @@
     "typescript": "^3.9.10"
   },
   "dependencies": {
+    "@serverless/utils": "pre-6",
     "fs-extra": "^7.0.1",
     "globby": "^10.0.2",
     "lodash": "^4.17.21"
   },
   "peerDependencies": {
-    "serverless": "2",
+    "serverless": "pre-3",
```

Comment:
I think we either should remove this setting (for eventual plugin pre-release case), or change it to `3` (assuming we want to merge and publish only after v3 is public).

`pre-3` cannot work as _peer_ setting, as it's  a _tag_ which always points a single publication (which changes with every push to v3 branch) and not a range of publications

Reply:
The goal would be:

- v2 users should be able to install a compatible version of this plugin
- v3 beta users should be able to install a compatible version of this plugin (that uses the new design)
- when v3 is released, v3 users should be able to install a compatible version of this plugin

How can we make this work?

We'll have this issue on every plugin.

Reply:
As in case of this plugin, it's that we drop support for Framework v2, I think it needs to go with pre-release now, and `master` should stick to support for v2.

Once Framework v3 will be released, we can then officially issue a new major for this plugin.

Reply:
ahhh ok I think I understand what you are trying to say: you are saying we release a beta version of this plugin, and v3-beta users have to install the beta version of this plugin explicitly?

Reply:
Yes, I think we cannot really release a public normal version, until there's a public normal version of Framework v3